prep-html-syntax/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <meta charset="UTF-8">
+
+  <title>Page Title</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link>
```

Comment:
Be sure to put your meta info into a `<head>` element

Reply:
Hi, I believe the changes have been made.

Thank you